reducer/src/test/java/com/graphicsfuzz/reducer/reductionopportunities/ReductionOpportunitiesTest.java

```diff
@@ -1125,4 +1125,27 @@ public void testCanRemoveLiveInjectedVariables() throws Exception {
     assertFalse(ops.isEmpty());
   }
 
+  @Test
+  public void testCanRemovePrecisionStatements() throws Exception {
+    final String shader = "#version 310 es\n"
+        + "precision highp float;\n"
+        + "precision highp int;\n"
+        + "precision mediump float;\n"
+        + "precision lowp int;\n"
+        + "precision highp int;\n"
+        + "precision highp float;\n"
+        + "void main() {\n"
+        + "}\n";
+
+    final TranslationUnit tu = ParseHelper.parse(shader);
+
+    List<IReductionOpportunity> ops = ReductionOpportunities.getReductionOpportunities(
+        MakeShaderJobFromFragmentShader.make(tu), new ReducerContext(false,
+            ShadingLanguageVersion.ESSL_310,
+            new RandomWrapper(0), new IdGenerator()), fileOps);
+    assertFalse(ops.isEmpty());
```

Comment:
Could remove this given that the following assertion is stronger than it.

Reply:
Done